client/manager_sql.go

```diff
@@ -179,10 +186,21 @@ func sqlDataFromClient(d *Client) (*sqlData, error) {
 		PostLogoutRedirectURIs:            strings.Join(d.PostLogoutRedirectURIs, "|"),
 		BackChannelLogoutURI:              d.BackChannelLogoutURI,
 		BackChannelLogoutSessionRequired:  d.BackChannelLogoutSessionRequired,
+		Metadata:                          string(metadata),
 	}, nil
 }
 
 func (d *sqlData) ToClient() (*Client, error) {
+	var metadata map[string]interface{}
+
+	if d.Metadata == "" {
```

Comment:
Can be omitted if change below is applied:

```suggestion
```

Reply:
Removed this code